pom.xml

```diff
@@ -45,8 +45,8 @@
     <dep.guava.version>18.0</dep.guava.version>
     <forkCount>2</forkCount>
     <parquet.version>1.10.0</parquet.version>
-    <calcite.version>1.16.0-drill-r7</calcite.version>
-    <avatica.version>1.11.0</avatica.version>
+    <calcite.version>1.17.0-drill-r0</calcite.version>
```

Comment:
What is the plan to start publishing Apache project artifacts to Apache maven repository instead of a proprietary one?

Reply:
I don't know the correct answer and how such cases of specific versions are handled in other Apache projects, so I may be wrong with my assumptions. 

Since this version contains Drill-specific commits and is used by Drill only, I'm not sure that it will be justified to publish it to the Apache maven repository.

Reply:
@vrozov, just to clarify for batch committers: Is this question blocks merging of this PR?

Reply:
Note for batch committer: no, the question is not a blocker for merging the PR.

@vvysotskyi I don't see why other projects will be confused by publishing an artifact that contains "drill" classifier. It sounds strange to me that an Apache project consumes an artifact produced by another Apache project not from the Apache repository.

Reply:
This version is built using the sources taken from our fork of Apache Calcite.

My expectation was if Apache maven repository contains some version of the project, then I can find the sources which were used for building the artifacts in the corresponding Apache repository.

Reply:
I don't know where the expectation for the source code in github comes from especially that github is not part of Apache INFRA, github Apache repositories are actually forks from Apache git and not all Apache projects use git as scm.

To clarify, it does not matter to me whether the artifact is published to Apache maven repository or another repository as long as it is published to a repository that is mirrored by maven central and the artifact is searchable in https://search.maven.org/

Reply:
@vrozov thanks for bringing up this point. Never thought about it but definitely valid one.
I have created Jira - https://issues.apache.org/jira/browse/DRILL-6711